### ISSUE TITLE
MJML App update with updated MJML engine

### ISSUE THREAD

**Issue by nvassos** [mjmlio/mjml-app#393]
Is there a way to use the new engine with the app? Any update on when a new MJML app would be released? MJML is on version 4.14 now.

**Comment by howdytom** [comment]
Yeah, I am also interested that. Unfortunately the MJML desktop versions still uses MJML v4.12. Do you have any plans to support the latest MJML engine version in the MJML desktop App?

**Comment by Marvin256** [comment]
Hello

Is this project dead?
